pasta/base/token_generator.py

```diff
@@ -32,13 +33,14 @@
 
 # Alias for extracting token names
 TOKENS = tokenize
+Token = collections.namedtuple('Token', ('type', 's', 'start', 'end', 'line'))
```

Comment:
maybe 'text'? 's' is a bit hard to read

Reply:
Ack, but I don't like text better, it's a string representation of anything (number, symbol, a string). Changed to "src" being as it stores a chunk of source code.